apps/user-profile/src/app/pages/identities-page/add-identity-button/add-identity-button.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { FocusOnLinkerDialogComponent, openLinkerWindow } from '@perun-web-apps/lib-linker';
+import {
+  NotificatorService,
+  OtherApplicationsService,
+  StoreService,
+} from '@perun-web-apps/perun/services';
+import { OAuthService } from 'angular-oauth2-oidc';
+import { TranslateService } from '@ngx-translate/core';
+import { getDefaultDialogConfig } from '@perun-web-apps/perun/utils';
+import { MatDialog } from '@angular/material/dialog';
+import { RegistrarManagerService } from '@perun-web-apps/perun/openapi';
+import { MatDialogRef } from '@angular/material/dialog/dialog-ref';
+
+@Component({
+  selector: 'perun-web-apps-add-identity-button',
+  templateUrl: './add-identity-button.component.html',
+  styleUrls: ['./add-identity-button.component.css'],
+})
+export class AddIdentityButtonComponent {
```

Comment:
Not sure why we would need a whole component for this button, feels a little strange.